### DISCUSSION TITLE
High memory usage on free tier - what to do?

### DISCUSSION THREAD

**Discussion by sunilguptasg** [supabase/supabase#11072] [Questions]
With just a handful of tables, less than 100 total rows of data, and one user - I see 82% memory usage on my free tier. What do I make of this? AFAIK, the free tier is on a 2-core ARM (shared) system with 1GB ram. 

Are all the shared users operating off the 1GB? 

I am not really happy with the performance even with one user - would I need to move to the pro plan even during development?

**Comment by mlanza** [comment]
It would be nice if it were possible momentarily upgrade to the next tier (e.g. for 2 hours) to test whether platform improvements actually solve memory and other tier limitation problems.  But, at that rate, I guess upgrading for a month, gets you the same albeit at a bit more cost.

**Comment by sunilguptasg** [comment]
From the response I received from Supabase support, I gather that the base paid tier is configured with the same hardware as the free tier.
